docs/source/onnx_dynamo.rst

```diff
@@ -0,0 +1,152 @@
+TorchDynamo-based ONNX Exporter
```

Comment:
Maybe add another section for exporting with dynamic shapes (not necessarily in this PR if you create an issue for tracking).

Reply:
You don't need to avoid duplicate code in examples. You can just copy-and-paste the current example and set a flag. For people reading example, they will have hard time if they can't just copy-and-paste to run it.

Reply:
This first PR is the opening for a series of examples on how to do stuff. I am also doing the same for the PyTorch tutorials page at https://github.com/pytorch/tutorials/pull/2541

As soon as we find the best structure for this doc, we can add sections for more advanced features, such as dynamic_shapes, fake mode, diagnostics, custom ops, etc